src/5e-SRD-StartingEquipment.json

```diff
@@ -1337,5 +1337,46 @@
       }
     ],
     "url": "/api/starting-equipment/wizard"
+  },
+  {
+    "index": "acolyte",
+    "background": {
+      "index": "acolyte",
+      "name": "Acolyte",
+      "url": "/api/backgrounds/acolyte"
+    },
+    "starting_equipment": [
+      {
+        "equipment": {
+          "index": "clothes-common",
+          "name": "Clothes, common",
+          "url": "/api/equipment/clothes-common"
+        },
+        "quantity": 1
+      },
+      {
+        "equipment": {
+          "index": "pouch",
+          "name": "Pouch",
+          "url": "/api/equipment/pouch"
+        },
+        "quantity": 1
+      }
+    ],
+    "starting_equipment_options": [
+      {
+        "choose": 1,
+        "type": "equipment",
+        "from": {
+          "equipment_category": {
+            "index": "holy-symbols",
+            "name": "Holy Symbols",
+            "url": "/api/equipment-categories/holy-symbols"
+          }
+        }
+      }
+
+    ],
+    "url": "/api/starting-equipment/wizard"
```

Comment:
Looks like you missed a spot! The URL should be `/api/starting-equipment/acolyte`.